cmd/README.md

```diff
@@ -17,13 +17,22 @@ For additional details, please refer to the [commatrix documentation](https://gi
 
 ---
 
-## Running
-build the code locally and install it in `/usr/local/bin`:
+## Install
+This plugin can be installed with the [Krew](https://sigs.k8s.io/krew) plugin manager:
+
+```sh
+wget https://github.com/openshift-kni/commatrix/blob/main/cmd/.krew.yaml
+oc krew install --manifest=.krew.yaml
```

Comment:
If we need to specify the path of the index file, what's the reason to start it with a dot?

Reply:
i saw that in regulare they are doing like that
but i will change now to be commatrix-krew.yaml